script/events/room.js

```diff
@@ -49,7 +49,37 @@ Events.Room = [
 				}
 			}
 		}
-	}, { /* Noises Outside  --  gain wood/fur */
+	}, 
+	{
+		title: _('Fire'),
+		isAvailable: function() { 
+			return $SM.get('game.buildings["hut"]', true) > 0 && $SM.get('game.population', true) > 5;
+		},
+		scenes: {
+			'start': {
+				text: [
+					_('A fire rampages through one of your huts, destroying it.'),

```

Comment:
Would be better as "a fire rampages through one of the huts, destroying it." ADR doesn't use personal pronouns or capital letters.